docs/book/how-to/setting-up-a-project-repository/stacks-pipelines-models.md

```diff
@@ -0,0 +1,107 @@
+- They set up a [repository for their project](../setting-up-a-project-repository/README.md) and start building their pipelines collaboratively. Let's assume Bob builds the feature engineering and training pipeline and Alice builds the inference pipeline.
+- To test their pipelines locally, they both have a `default` stack with a local orchestrator and a local artifact store. This allows them to quickly iterate on their code without deploying any infrastructure or incurring any costs.
+- While building the inference pipeline, Alice needs to make sure that the preprocessing step in her pipeline is the same as the one used while training. It might even involve the use of libraries that are not publicily available and she follows the [Shared Libraries and Logic for Teams](./shared_components_for_teams.md) guide to help with this.
+- Bob's training pipeline produces a model artifact, which Alice's inference pipeline requires as input. It also produces other artifacts such as metrics and a model checkpoint that are logged as artifacts in the pipeline run.
+- To allow easy access to model and data artifacts, they [use a ZenML Model](../../how-to/use-the-model-control-plane/associate-a-pipeline-with-a-model.md) which ties the pipelines, models and artifacts together. Now Alice can just [reference the right model name and find the model artifact she needs.](../../how-to/use-the-model-control-plane/load-artifacts-from-model.md)
+- It is also critical that the right model version from the training pipeline is used in the inference pipeline. The [Model Control Plane](../../how-to/use-the-model-control-plane/README.md) helps Bob to keep track of the different versions and to easily compare them. Bob can then [promote the best performing model version to the `production` stage](../../how-to/use-the-model-control-plane/promote-a-model.md) which Alice's pipeline can then consume.
+- Alice's inference pipeline produces a new artifact, in this case a new dataset containing the predictions of the model. Results can also be added as metadata to the model version, allowing easy comparisons.
+
+This is a very simple example, but it shows how you can use ZenML to structure your ML workflow. You can use the same principles for more complex workflows. 
+
+## Rules of Thumb
+
+Here are some general guidelines to help you organize your ZenML projects effectively:
+
+### Models
+- Create one Model per distinct machine learning use-case or business problem
+- Use Models to group related pipelines, artifacts, and metadata together
+- Leverage the Model Control Plane to manage model versions and stages (e.g., staging, production)
+
+### Stacks
+- Maintain separate stacks for different environments (development, staging, production)
+- Share production and staging stacks across teams to ensure consistency
+- Keep local development stacks simple for quick iterations
+
+### Organization
```

Comment:
This might be confused with a ZenML Pro Organization

Reply:
updated